src/jvmTest/resources/test.sarif.json

```diff
@@ -21,6 +21,14 @@
                   "startColumn": 1,
                   "startLine": 1
                 }
+              },
+              "properties": {
```

Comment:
Is there still a test where there's no property?

Reply:
All the classes has `properties` so I would say yes. But as I said before, this repo needs way more tests.

Reply:
ah, true, cool